prow/crier/reporters/gerrit/reporter.go

```diff
@@ -391,7 +391,11 @@ func GenerateReport(pjs []*v1.ProwJob) JobReport {
 
 	}
 	report.Header = defaultProwHeader
-	report.Header += fmt.Sprintf(" %d out of %d pjs passed!\n", report.Success, report.Total)
+	var reTestMessage string
+	if report.Success < report.Total {
+		reTestMessage = " Say '/retest' to rerun all failed tests"
```

Comment:
`Comment` instead of `Say`?

Reply:
Also I wonder if we can also add a link to the prow help page (gerrit version of https://prow.k8s.io/command-help)?

Reply:
Done with `Comment`.

`command-help` is way more complex, would be a separate feature request, and probably worth a discussion around whether we want to consolidate gerrit `/retest` with all other plugins